packages/atclient/src/connection.c

```diff
@@ -841,12 +841,6 @@ static int atclient_connection_set_port(atclient_connection *ctx, const uint16_t
     return ret;
   }
 
-  if (port < 0) {
```

Comment:
port was changed to an unsigned 16 bit integer, so it can never be less than zero.